index.html

```diff
@@ -0,0 +1,165 @@
+          facere quae, dolore perspiciatis iste sapiente ut a animi! Voluptate
+        </p>
+      </div>
+      <div id="time">
+        <h2>2020.10.15(THU) ~ 16(FRI)</h2>
+        <h6>@National Museum of Rwanda, Art Center</h6>
+      </div>
+    </div>
+
+    <section class="main-program">
+      <h2>Main Programs</h2>
+              <hr>
+      <div class="program-list">
+        <div class="programs">
+          <div class="p-icon"><a href="#"><img src="./Images/program_icon_01.png" alt=""></a></div>
+          <div class="p-header">
+            <h3>Lecture</h3>
+          </div>
+          <div class="p-paragraph">
+            <p>Appreciate  various creations applying CC license of artists, organized from Art Center Nabi.</p>
+          </div>
+        </div>
+        <div class="programs">
+          <div class="p-icon"><a href="#"><img src="./Images/program_icon_01.png" alt=""></a></div>
+          <div class="p-header">
+            <h3>Lecture</h3>
+          </div>
+          <div class="p-paragraph">
+            <p>Appreciate  various creations applying CC license of artists, organized from Art Center Nabi.</p>
+          </div>
+        </div>
+        <div class="programs">
+          <div class="p-icon"><a href="#"><img src="./Images/program_icon_01.png" alt=""></a></div>
+          <div class="p-header">
+            <h3>Lecture</h3>
+          </div>
+          <div class="p-paragraph">
+            <p>Have time to share your thoughts and opinions with experts for each topic.</p>
+          </div>
+        </div>
+        <div class="programs">
+          <div class="p-icon"><a href="#"><img src="./Images/program_icon_04.png" alt=""></a></div>
+          <div class="p-header">
+            <h3>Lecture</h3>
+          </div>
+          <div class="p-paragraph">
+            <p>Try creating your own work using open source license rather than just watching.</p>
+          </div>
+        </div>
+        <div class="programs">
+          <div class="p-icon"><a href="#"><img src="./Images/program_icon_02.png" alt=""></a></div>
+          <div class="p-header">
+            <h3>Legend</h3>
+          </div>
+          <div class="p-paragraph  p-space">
+            <p>Get opportunities to work wit CC affiliates around the world, also after the summit.</p>
+          </div>
+        </div>
+      </div>
+      <div class="see-the-program">
+        <a href="#">Checkout all the programs</a>
+      </div>
+    </section>
+    <div id="featured-speakers">
+      <h2>Featured Speakers</h2>
+      <hr>
+    </div>
+    <section id="speakers" >
+        
+    </section>
+
+    <footer>
+      <div id="footer-div">
+        <h2>Partners</h2>
+         <hr>
+         <div class="footer-images">
+         <img src="./Images/apple-logo'.png" alt="">
+         <img src="./Images/android-logo-0.png" alt="">
+         <img src="./Images/airbnb.png" alt="">
+        </div>
+    </footer>
```

Comment:
- [ ] **As requested by the previous reviewe**r, I think it would be better to use the same background color to match the give design :+1: 

**Your design**
![image](https://user-images.githubusercontent.com/102757126/204101446-fa8214c6-4618-4624-a357-05ac1e0561e0.png)
**Design to follow**
![image](https://user-images.githubusercontent.com/102757126/204101477-65073873-e26d-411b-9c8e-c11e5546138b.png)